### PR TITLE
Alert & button

### PR DESCRIPTION
**Pull request by amartya-rs**
feature update: alert & button component

merge 2

### CODE REVIEW

File: all-components/alert.css [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+/*common alert styling*/
+.alert {
+    font-weight: bold;
+    padding: 0.5rem 1rem;
+    height: 3rem;
+    width: 80%;
+    border-radius: 0.65rem;
+    
+
```
Comment (zoef-yt) [inline all-components/alert.css]: extra spacing can be removed

File: all-components/button.css [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+/*button with icons*/
+.icon {
+    height: 1.25rem;
+    width: 1.25rem;
+}
+
+.button-icons {
+    min-width: max-content;
+    gap: 0.5rem;
```
Comment (zoef-yt) [inline all-components/button.css]: Change this to 5px

**Review by zoef-yt — CHANGES_REQUESTED** [review]

File: all-components/alert.css (hunk 1 above)
Comment (iRohitGaur) [inline all-components/alert.css:12]: remove empty spacing

**Review by iRohitGaur — APPROVED** [review]
Nice work! Good to merge

File: all-components/alert.css (hunk 1 above)
Comment (amartya-rs) [reply all-components/alert.css]: Done.

File: all-components/button.css (hunk 2 above)
Comment (amartya-rs) [reply all-components/button.css]: My root font size is the default 16px, so 0.5rem will be 8px. I have updated gap with 8px.

File: all-components/alert.css (hunk 1 above)
Comment (amartya-rs) [reply all-components/alert.css:12]: Done.

**Review by sudipkundu999 — APPROVED** [review]
Just take care of the extra spacing at some places,
Good class names! Good to merge.

**Comment by amartya-rs** [comment]
> Just take care of the extra spacing at some places, Good class names! Good to merge.

Thanks Sudip for your feedback, will keep this point in mind.

**Comment by archit7448** [comment]
Meaning full class names ,only spacing can remove .Nice work good to merge

**Review by archit7448 — APPROVED** [review]

**Review by zoef-yt — APPROVED** [review]

**Comment by amartya-rs** [comment]
> Meaning full class names ,only spacing can remove .Nice work good to merge

Thanks Archit for your feedback.

**Pull request merged**